### PR TITLE
i18n(es): Translated `from-nextjs.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `from-nextjs.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+
+Alternativamente, puedes usar [colecciones de contenido](/es/guides/content-collections/) en Astro para almacenar y administrar tu contenido. Cuando forman parte de una colección, los archivos Markdown y MDX vivirán en carpetas dentro de `src/content/`.  Deberás recuperar el contenido tú mismo y [generar esas páginas dinámicamente](/es/guides/content-collections/#generating-routes-from-content).
+
+### Migrando Tests
+
+Como Astro produce HTML sin procesar, es posible escribir tests end-to-end utilizando la salida del paso de compilación. Cualquier test end-to-end escrito anteriormente  podría funcionar sin problemas si has podido coincidir con el marcado de tu sitio de Next. Bibliotecas de testing como Jest y React Testing Library pueden ser importadas y utilizadas en Astro para testear tus componentes de React.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx:321]:
```suggestion
Como Astro produce HTML sin procesar, es posible escribir tests end-to-end utilizando la salida del paso de compilación. Cualquier test end-to-end escrito anteriormente podría funcionar sin problemas si has podido coincidir con el marcado de tu sitio de Next. Bibliotecas de testing como Jest y React Testing Library pueden ser importadas y utilizadas en Astro para poner a prueba tus componentes de React.
```

File: src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+
+Estas [páginas `.astro`](/es/core-concepts/astro-pages/) deben estar ubicadas dentro de `src/pages/`  sus rutas de página se generarán automáticamente en función de su ruta de archivo.
+
+#### Páginas de Markdown y MDX
+
+Astro tiene soporte incorporado para Markdown y una integración opcional para archivos MDX. Puedes reutilizar cualquier archivo [Markdown y MDX](/es/guides/markdown-content/) existente, pero pueden requerir algunos ajustes en su frontmatter, como agregar [la propiedad especial de frontmatter `layout`de Astro](/es/core-concepts/layouts/#plantillas-de-markdownmdx). Ya no necesitarás crear manualmente páginas para cada ruta generada por Markdown. Estos archivos se pueden colocar dentro de`src/pages/` para aprovechar el enrutamiento automático basado en archivos.
+
+Alternativamente, puedes usar [colecciones de contenido](/es/guides/content-collections/) en Astro para almacenar y administrar tu contenido. Cuando forman parte de una colección, los archivos Markdown y MDX vivirán en carpetas dentro de `src/content/`.  Deberás recuperar el contenido tú mismo y [generar esas páginas dinámicamente](/es/guides/content-collections/#generating-routes-from-content).
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx:317]:
```suggestion
Alternativamente, puedes usar [colecciones de contenido](/es/guides/content-collections/) en Astro para almacenar y administrar tu contenido. Cuando forman parte de una colección, los archivos Markdown y MDX vivirán en carpetas dentro de `src/content/`. Deberás recuperar el contenido tú mismo y [generar esas páginas dinámicamente](/es/guides/content-collections/#generating-routes-from-content).
```

File: src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+
+Puede resultarte útil comenzar convirtiendo tus diseños y plantillas de Next.js en [componentes de diseño de Astro](/es/core-concepts/layouts/).
+
+Next tiene dos métodos diferentes para crear archivos de diseño, cada uno de los cuales maneja los diseños de manera diferente a Astro:
+
+- El directorio `pages`
+
+- [El directorio `/app` (en beta)](https://beta.nextjs.org/docs/routing/pages-and-layouts#layouts)
+
+Cada página de Astro requiere explícitamente que se incluyan las etiquetas `<html>`, `<head>`, y `<body>`, por lo que es comñun reutilizar un archivo de diseño en varias páginas. Astro utiliza [`<slot />`](/es/core-concepts/astro-components/#slots) para el contenido de la página, sin necesidad de importar ninguna declaración. Observa la estructura estándar de templating de HTML y el acceso directo al elemento `<head>`:
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-nextjs.mdx:202]:
```suggestion
Cada página de Astro requiere explícitamente que se incluyan las etiquetas `<html>`, `<head>`, y `<body>`, por lo que es común reutilizar un archivo de diseño en varias páginas. Astro utiliza [`<slot />`](/es/core-concepts/astro-components/#slots) para el contenido de la página, sin necesidad de importar ninguna declaración. Observa la estructura estándar de templating de HTML y el acceso directo al elemento `<head>`:  
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo algunas sugerencias 😄

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @Waxer59 & @dreyfus92 🙌

**Pull request merged**